src/Widgets/Text.cpp

```diff
@@ -85,7 +99,18 @@ void Text::SetColor(const SDL_Color &newColor) {
   color.g = newColor.g;
   color.b = newColor.b;
   color.a = newColor.a;
-  if (_textContent != "") {
-    SetTextContentTexture();
+  SetTextContentTexture();
+}
+
+void Text::SetWrapLength(Uint32 wrapLength) {
+  _wrapLength = wrapLength;
+  SetTextContentTexture();
+}
+
+Uint32 Text::GetWrapLength() const {
```

Comment:
Won't this set the width to the parent? What if you still want to center the text? This forces all text to always be left aligned.